pom.xml

```diff
@@ -85,7 +85,7 @@
 
         <project.encoding>UTF-8</project.encoding>
         <slf4j.version>1.7.21</slf4j.version>
-        <sofa.common.tools>1.0.12</sofa.common.tools>
+        <sofa.common.tools>1.4.0</sofa.common.tools>
```

Comment:
this version does not exist in https://mvnrepository.com/artifact/com.alipay.sofa.common/sofa-common-tools

Reply:
sofa-common-tools 1.4.0 has been released